packages/fast-components-styles-msft/src/utilities/high-contrast.ts

```diff
@@ -16,18 +16,22 @@ export enum HighContrastColor {
     background = "Background",
 }
 
+function ImportantColor(color: HighContrastColor): string {
+    return color + "!important";
```

Comment:
no space between the color and !important, also prefer using back ticks here.

Reply:
updated with back ticks